src/scripts/hunt-filter/messageExemptions.ts

```diff
@@ -0,0 +1,31 @@
+import { IntakeMessage } from "../types/mhct";
```

Comment:
Make sure type imports use `import type`

Reply:
Could you expound on that? I'm not sure what you mean.

Reply:
@tehhowch ^^

Reply:
The only valid way here to use `import IntakeMessage from "../types/mhct"` is if IntakeMessage is a `default export` from mhct, but it is not so this is the only valid notation for importing.